### ISSUE TITLE
livequery

### ISSUE THREAD

**Issue by julien9999** [timanrebel/Parse#53]
Hi, is livequery implemented?
if yes, how to use it ?
thanks

**Comment by timanrebel** [comment]
No, this module currently only implemented push notifications for Android. I always used the Alloy models with Parse, which worked perfectly. Maybe the Parse JS SDK works as well?

**Comment by ULSashido** [comment]
LiveQuery and other goodies: https://github.com/hansemannn/titanium-parse-live-query
